setup.cfg

```diff
@@ -9,17 +9,6 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
-[nosetests]
```

Comment:
nose was use for coverage you can not remove this like this. This is probably why the coveralls status is shown as waiting for result.

Also it would be good to make sure that doctests are tested.